masterlist.yaml

```diff
@@ -2895,6 +2895,45 @@ plugins:
       - Delev
       - Invent
       - Relev
+  - name: 'TheWulfPanda - City Teleport Spells.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/7015/'
+        name: 'Teleport - City Spells on Nexus Mods'
+    inc:
+      - 'TheWulfPanda - City Teleport Spells Less Magicka Increase Cast Time.esp'
+    dirty:
+      # v1.01
+      - crc: 0xEC89CAD3
+        <<: *dirtyPlugin
+        itm: 19
+        udr: 0
+        nav: 0
+    clean:
+      # v1.01 ~ remove wild edits after cleaning:
+      #   - parent sub-block of 0000939D
+      #   - 00042249
+      #   - 00020EE7
+      #   - 0001A27F
+      #   - 00037EE9
```

Comment:
Also you don't have to link to a guide for the wild edits. You can do something like below.

```
    msg:
      - <<: *hasWildEdits
        subs:
          - 'TheWulfPanda - City Teleport Spells.esp'
          - 'Remove the following records with xEdit: MorthalExterior04 [CELL:0000939D],SolitudeOrigin [CELL:00037EE9],WhiterunOrigin [CELL:0001A27F],MarkarthOrigin [CELL:00020EE7],RiftenCityNorth [CELL:00042249]'
        condition: 'checksum("TheWulfPanda - City Teleport Spells.esp",F4169312)'
```

Reply:
So you can duplicate that message in the other entry and replace the esp name and CRC.

Reply:
Does the LOOT messaging system support line breaks or will they be ignored?
If we can use line breaks then I think something like this would be a bit more readable in the code as well as in the frontend:

```yaml
    msg:
      - <<: *hasWildEdits
        subs:
          - 'TheWulfPanda - City Teleport Spells.esp'
          - |
            Remove the following records with xEdit:
              - Parent Sub-Block 2,-2 of MorthalExterior04 [CELL:0000939D]
              - SolitudeOrigin [CELL:00037EE9]
              - WhiterunOrigin [CELL:0001A27F]
              - MarkarthOrigin [CELL:00020EE7]
              - RiftenCityNorth [CELL:00042249]
        condition: 'checksum("TheWulfPanda - City Teleport Spells.esp",F4169312)'
```

Reply:
I don't think it does, you would still need to wrap the string in Apostrophes ' '

Reply:
YAML supports two types of multi-line strings, the pipe `|` and the greater than sign `>`.

Using `>` is actually the same as wrapping the string in quotes, but you can use line breaks to make the code more readable. The line breaks will be ignored and replaced with a whitespace.

**single-line:**
```yaml
my_message: 'line 1 line 2 line 3'
```

**pseudo multi-line:**
```yaml
my_message: >
  line 1
  line 2

  line 3
```

Both versions above will have the same **output**:
> line 1 line 2 line 3

But the `|` will actually pass the line breaks to the application.

**real multi-line:**
```yaml
my_message: |
  line 1
  line 2

  line 3
```
**output:**
> line 1
> line 2
>
> line 3

But to be able to render those multi-line strings, the application has to translate them into valid HTML.

I just looked into the [source code](https://github.com/loot/loot/blob/dev/src/gui/html/elements/loot-plugin-card.js#L230) and LOOT even supports [Markdown](https://guides.github.com/features/mastering-markdown/) to format the messages. So the example in my last comment would look something like this in LOOT:

> Remove the following records with xEdit:
>   - Parent Sub-Block 2,-2 of MorthalExterior04 [CELL:0000939D]
>   - SolitudeOrigin [CELL:00037EE9]
>   - WhiterunOrigin [CELL:0001A27F]
>   - MarkarthOrigin [CELL:00020EE7]
>   - RiftenCityNorth [CELL:00042249]

Reply:
Yeah it will work, it has been used in the past for common entries in the skyrim masterlist. 

```
    msg:
      - <<: *hasWildEdits
        subs:
          - 'TheWulfPanda - City Teleport Spells.esp'
          - |

            > 
            Remove the following records with xEdit:
            ```
              Parent Sub-Block 2,-2 of MorthalExterior04 [CELL:0000939D]
              SolitudeOrigin [CELL:00037EE9]
              WhiterunOrigin [CELL:0001A27F]
              MarkarthOrigin [CELL:00020EE7]
              RiftenCityNorth [CELL:00042249]
            ```
```
Which would get you:
![image](https://user-images.githubusercontent.com/1944639/42122937-89b8688a-7c41-11e8-8c60-90f59cebdd5f.png)

HTML looks useful.
```
    msg:
      - <<: *hasWildEdits
        subs:
          - 'TheWulfPanda - City Teleport Spells.esp'
          - |
            <p>
              <h3>Remove the following records with xEdit:</h3>
                <ul>
                  <li>Parent Sub-Block 2,-2 of MorthalExterior04 [CELL:0000939D]</li>
                  <li>SolitudeOrigin [CELL:00037EE9]</li>
                  <li>WhiterunOrigin [CELL:0001A27F]</li>
                  <li>MarkarthOrigin [CELL:00020EE7]</li>
                  <li>RiftenCityNorth [CELL:00042249]</li>
                </ul>
            </p>
```
![image](https://user-images.githubusercontent.com/1944639/42122975-d18a9bce-7c41-11e8-9c4b-26fab4a7d948.png)

@WrinklyNinja Is the use of  `|` and `>` okay?

Reply:
Yes, you can use `|` and `>` to make the masterlist more readable, and `|` is preferable to using `\n`. I'd avoid using HTML as Markdown covers most text formatting options and is more readable in source form.

Reply:
Okay so the above that @jb29 suggested is fine?

```yaml
    msg:
      - <<: *hasWildEdits
        subs:
          - 'TheWulfPanda - City Teleport Spells.esp'
          - |
            Remove the following records with xEdit:
              - Parent Sub-Block 2,-2 of MorthalExterior04 [CELL:0000939D]
              - SolitudeOrigin [CELL:00037EE9]
              - WhiterunOrigin [CELL:0001A27F]
              - MarkarthOrigin [CELL:00020EE7]
              - RiftenCityNorth [CELL:00042249]
        condition: 'checksum("TheWulfPanda - City Teleport Spells.esp",F4169312)'
```

Reply:
Yes, that's fine.